### DISCUSSION TITLE
How to visualize the results of local_thickness on a 3D image?

### DISCUSSION THREAD

**Discussion by josephinoo** [PMEAL/porespy#491] [Q&A] [answered]
Hello, could you help me, how to graph the Local thickness in 3D, I read a paper of your property, that they used that method, I would like to be able to graph with the image shown below: 
![Captura de Pantalla 2021-04-14 a la(s) 23 43 34](https://user-images.githubusercontent.com/46098690/114815362-a9308680-9d7b-11eb-8e70-cdd6ca9cb094.png)

**Comment by jgostick** [comment]
We used paraview for that.  You can save images as tiffs, and opening them directly in paraview. You then use a 'threshold' to remove parts you don't want.  also, be sure to enable ray tracing to get the photorealistic shadows ;-)

**Comment by josephinoo** [comment]
Could you help me with an example to guide me, I have created the tiff with tifffile, but it only shows a slice in paraview 
<img width="1264" alt="Captura de Pantalla 2021-04-15 a la(s) 00 28 29" src="https://user-images.githubusercontent.com/46098690/114818512-8608d580-9d81-11eb-8a68-f1fb88fd34e4.png">

**Comment by jgostick** [comment, marked as answer]
You probably only saved 1 slice.  Use the imageio package:  ``imageio.volsave('filename.tif', np.array(tif_array, dtype=np.int32)``

**Reply by josephinoo** [reply]
Try this, to generate it:

```python
blobs = ps.generators.blobs(shape=[200,200,200] porosity=0.5, blobiness=1)
thk = ps.filters.local_thickness(blobs, mode='dt')
a=np.array(thk, dtype=np.float32)
tif.imsave('BLOB1.tif', a, bigtiff=True)
```

![blob1](https://user-images.githubusercontent.com/46098690/115127308-27b73f00-9f9b-11eb-91c4-d2fd218e31fc.png)

Thank you very much for the recommendations

**Reply by jgostick** [reply]
that looks very pretty! Well done.
